emperor/support_files/js/axes-controller.js

```diff
@@ -8,7 +8,7 @@ define([
 ], function($, _, DecompositionView, ViewControllers, d3, contextmenu) {
 
   // we only use the base attribute class, no need to get the base class
```

Comment:
You are right, this is out of date.

On (Jul-28-16| 8:34), Daniel McDonald wrote:
> > @@ -8,7 +8,7 @@ define([
> >  ], function($, _, DecompositionView, ViewControllers, d3, contextmenu) {
> >  
> >    // we only use the base attribute class, no need to get the base class
> 
> is this comment still accurate?
> 
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/528/files/6b118f7f6dd9154c17ff1dfb19d2ad7b90b32b97#r72645917